signup/tool/src/java/org/sakaiproject/signup/tool/jsf/SignupUIBaseBean.java

```diff
@@ -699,6 +699,14 @@ public String generateGroupDescription(String meetingTitle, SignupTimeslot times
 		return uuids;
 	}
 
+	/**
+	 * Determines if user has a stale session
```

Comment:
The term stale session is causing confusion and so maybe something along the lines of `isSessionGood()` or `isSessionValid()` or `hasMeetingWrapper`?